<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewConnectionDialog</class>
 <widget class="QDialog" name="NewConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Template Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabIdentificationDialog">
      <attribute name="title">
       <string>Identification</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="1">
            <widget class="QTextEdit" name="description_le">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Detailed multi-line description to fully explain the collection.</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="name_le">
             <property name="toolTip">
              <string>Identifier for the collection that is unique across the provider.</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="name_la">
             <property name="toolTip">
              <string>Identifier for the collection that is unique across the provider.</string>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="title_le">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>A short descriptive one-line title for the collection.</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="title_la">
             <property name="toolTip">
              <string>A short descriptive one-line title for the collection.</string>
             </property>
             <property name="text">
              <string>Title</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="description_la">
             <property name="toolTip">
              <string>Detailed multi-line description to fully explain the collection.</string>
             </property>
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="image_la">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../../../Pictures/templates_sample.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="toolTip">
            <string>Download project used to create this template</string>
           </property>
           <property name="text">
            <string>Download project</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="open_layout_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Open templates inside the QGIS layout manager</string>
           </property>
           <property name="text">
            <string>Open in Layout Manager</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabKeywordsDialog">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="extension_la">
         <property name="toolTip">
          <string>Identifier for the collection that is unique across the provider.</string>
         </property>
         <property name="text">
          <string>Template extension</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="extension_le">
         <property name="toolTip">
          <string>Identifier for the collection that is unique across the provider.</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="template_type_la">
         <property name="toolTip">
          <string>A short descriptive one-line title for the collection.</string>
         </property>
         <property name="text">
          <string>Template type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="template_type_le">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>A short descriptive one-line title for the collection.</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="thumbnail_la">
         <property name="toolTip">
          <string>A short descriptive one-line title for the collection.</string>
         </property>
         <property name="text">
          <string>Thumbnail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="thumbnail_le">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>A short descriptive one-line title for the collection.</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="license_la">
         <property name="toolTip">
          <string>Detailed multi-line description to fully explain the collection.</string>
         </property>
         <property name="text">
          <string>License</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="license_le">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Detailed multi-line description to fully explain the collection.</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabExtentDialog">
      <attribute name="title">
       <string>Resources</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Template spatial and  temporal extent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsExtentGroupBox" name="spatialExtentSelector">
         <property name="title">
          <string>Spatial extent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
         <property name="title">
          <string>Temporal extent</string>
         </property>
         <property name="collapsed">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>From</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsDateTimeEdit" name="from_date"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_34">
              <property name="text">
               <string>To</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsDateTimeEdit" name="to_date"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>qgsdatetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
